project/ws/app/src/lib/routes/home/components/search/search.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="flex flex-1 flex-row margin-top-m margin-bottom-m">
+    <div class="search flex margin-right-m" [ngClass]="{'disable': isContentLive}">
+        <div class="rsearch flex w-full"><mat-icon
+                class="color-60 search-icon mat-icon notranslate material-icons mat-icon-no-color">search</mat-icon>
+            <input class="sinput color-60 w-full" [(ngModel)]="searchText" (keyup.enter)="searchData($event)" label=""
+                placeholder="Select" type="Standard" [ngClass]="{'disable': isContentLive}">
+        </div>
+    </div>
+    <div class="filter-section flex">
+        <button mat-button type="button" (click)="sort()" class="apply-filter margin-right-s">
+            <mat-icon class="margin-right-xs">sort</mat-icon> Sort
+        </button>
+    </div>
+    <div class="filter-section flex">
+        <button mat-button type="button" (click)="openFilter()" class="apply-filter margin-right-s">
+            Apply Filter <mat-icon class="margin-right-xs">filter_list</mat-icon>
```

Comment:
same as above

Reply:
Filter is required here & have used which is required only